app/views/coronavirus_landing_page/components/landing_page/_nhs_notice.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  # hardcoding data that is expected to be in the Content Item
+  nhs_banner = {
+    "sections" => [
+      {
+        "heading" => "If you have no symptoms:",
+        "markdown" => "Check if you should get [regular rapid lateral flow tests](/getting-tested-for-coronavirus).",
+      },
+      {
+        "heading" => "If you have coronavirus symptoms:",
+        "markdown" => "- [get a PCR test](/get-coronavirus-test)\n- stay at home",
+      }
+    ]
+  }
+%>
+<section class="covid__nhs-notice"
+         aria-label="notice"
+         role="region"
+         data-module="track-links"
+         data-track-category="pageElementInteraction"
+         data-track-action="callOutBox">
+  <div class="covid__nhs-notice__logo">
+    <%= render "govuk_publishing_components/components/organisation_logo", {
+      organisation: {
+        name: "nhs",
+        url: "https://www.nhs.uk/",
+        image: {
+          url: image_url("nhs-logo.svg"),
+          alt_text: "NHS"
+        }
+      }
+    } %>
+  </div>
+
+  <% nhs_banner["sections"].each do |section| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: section["heading"],
+      heading_level: 2,
+      font_size: "m",
+      margin_bottom: 4,
+    } if section["heading"].present? %>
+
+    <%= render_govspeak(section["markdown"]) if section["markdown"].present? %>
```

Comment:
What will happen if the markdown is invalid? I tried to break it and found that it can spill over the box quite easily... would some sort of a line length check work?

<img width="453" alt="markdown" src="https://user-images.githubusercontent.com/5963488/112856896-2f747980-90a8-11eb-8ce5-2dd0174146b7.png">

Reply:
Thanks for pointing this out. I see this as a concern for someone editing the copy rather than a code concern, it's an existing possibility for any of the currently editable fields of the existing approach, rather than something introduced in this change.

Reply:
I think it'll be unlikely that we'd have such a long word but we could add a `word-break: break-word;` css for good measure.

Reply:
👍  I suppose unless we actually put this into the publishing tool then its going to be difficult to validate.